front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1699,6 +1711,46 @@ export type GetV2TimetableByIdConflictsApiArg = {
   id: number;
   infraId: number;
 };
+export type PostV2TimetableByIdStdcmApiResponse =
+  /** status 201 The simulation result */
+  | {
+      departure_time: string;
+      path: PathfindingResultSuccess;
+      simulation: SimulationResponse;
+      status: 'success';
+    }
+  | {
+      status: 'path_not_found';
+    };
+export type PostV2TimetableByIdStdcmApiArg = {
+  /** The infra id */
+  infra: number;
+  /** timetable_id */
+  id: number;
+  body: {
+    comfort: Comfort;
+    margin?: MarginValue | null;
+    /** By how long we can shift the departure time in milliseconds */
+    maximum_departure_delay?: number;
+    /** Specifies how long the total run time can be in milliseconds */
+    maximum_run_time?: number;
```

Comment:
I don't see a `maximum_arrival_delay`, does this one refers to it ?

Reply:
`maximum_run_time` refers to the maximum time core is allowed to run the simulation.
In STDCM V1, core did not use `maximum_arrival_delay`. This is still the case in STDCM V2. That's why it disappears.

Reply:
Ok but on the new stdcm sketch, we still can chose a tolerance for the destination : 

![image](https://github.com/OpenRailAssociation/osrd/assets/113104137/ec9ed80f-cc8f-4fec-8378-13a1f1e7275b)

It won't be implemented yet ?